integration-cli/docker_api_containers_test.go

```diff
@@ -176,7 +176,7 @@ func (s *DockerSuite) TestContainerApiStartVolumesFrom(c *check.C) {
 		c.Fatal(out, err)
 	}
 
-	name := "testing"
+	name := "testing" + stringid.GenerateRandomID()
```

Comment:
our convention usually is to use the test's name: `name := "testcontainerapistartvolumesfrom"` and btw I'd much rather have it named exactly like the test: TestContainerApiStartVolumesFrom, but that's not how we've done it so far.